### ISSUE TITLE
Test expects runtime type of a Type object to be a subtype of the Type 

### ISSUE THREAD

**Issue by a-siva** [dart-lang/co19#487]
The test co19_2/src/LanguageFeatures/Extension-methods/extension_member_resolution_t04.dart

Has code as follows:

```
class A {}
class C extends A {}

extension Ext<T extends A> on List<T> {
  Type method() => T;
}

main() {
  List<C> list1 = [];
  Expect.isTrue(list1.method() is C);

  List<A> list2 = [];
  Expect.isTrue(list2.method() is A);
}
```

The is incorrect, instead the test could do something like

```
  Expect.isTrue(list1.method() == C);
  Expect.isTrue(list2.method() == A);

```

**Comment by a-siva** [comment]
co19_2/src/LanguageFeatures/Extension-methods/extension_member_resolution_t04.dart has a similar problem, the code should be
```
Expect.isTrue(list1.method() == C);
Expect.isTrue(list2.method() == A);

```

**Comment by sgrekhov** [comment]
Duplicate of #480. Already fixed by  cc98d34
